### PR TITLE
peopleEatChoice

### PR DESCRIPTION
**Pull request by Llavrov**

### CODE REVIEW

File: src/components/standard/icon/PeopleEatIconProps.ts [hunk 1]
```diff
@@ -4,4 +4,5 @@ export interface PeopleEatIconProps {
     className?: string;
     edgeLength?: number;
     icon: Icon;
+    onClick?: () => void;
```
Comment (CemYil03) [inline src/components/standard/icon/PeopleEatIconProps.ts:7]: Please remove this, the icon component should not have an onClick callback, only it's parent

File: src/components/standard/editButton/PeopleEatEditButton.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+const meta: Meta<typeof PeopleEatEditButton> = {
+    title: 'Standard Components/PeopleEatEditButton',
+    component: PeopleEatEditButton,
+};
+
+export default meta;
+
+export const Component: StoryObj<typeof PeopleEatEditButton> = {
+    args: {
+        onClick: () => console.log('click'),
```
Comment (CemYil03) [inline src/components/standard/editButton/PeopleEatEditButton.stories.tsx:13]: Please remove all console log's

File: public/Mastercard.png
Comment (CemYil03) [inline public/Mastercard.png:1]: Can you please name it lower case Mastercard -> master-card

File: src/components/standard/changePhotoButton/changePhotoButton.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { type ReactElement } from 'react';
+import { Icon } from '~/components/standard/icon/Icon';
+import PeopleEatIcon from '~/components/standard/icon/PeopleEatIcon';
```
Comment (CemYil03) [inline src/components/standard/changePhotoButton/changePhotoButton.tsx:3]: Please remove all tilde imports and replace them with a relative path

**Review by CemYil03 — CHANGES_REQUESTED** [review]

File: src/components/standard/icon/PeopleEatIconProps.ts (hunk 1 above)
Comment (Llavrov) [reply src/components/standard/icon/PeopleEatIconProps.ts:7]: okay

File: src/styles/globals.css [hunk 4]
```diff
@@ -1,3 +1,11 @@
...
 @tailwind components;
 @tailwind utilities;
+
+@import url('https://fonts.googleapis.com/css2?family=Manrope:wght@200;300;400;500;600;700&display=swap');
+
+@layer base {
+    html {
+        font-family: Manrope, sans-serif;
+    }
+}
```
[the remark this answers is not in the document]
Comment (Llavrov) [reply src/styles/globals.css:11]: to set font-family for all components

File: tailwind.config.cjs [hunk 5]
```diff
@@ -1,6 +1,9 @@
 /** @type {import('tailwindcss').Config} */
 const config = {
     content: ['./src/**/*.{js,ts,jsx,tsx}'],
+    corePlugins: {
+        preflight: false,
+    },
```
[the remark this answers is not in the document]
Comment (Llavrov) [reply tailwind.config.cjs:6]: Don't know exactly, but want to try, saw this solution in documentation

File: src/components/standard/choice/PeopleEatChoice.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+const meta: Meta<typeof PeopleEatChoice> = {
+    title: 'Standard Components/PeopleEatChoice',
+    component: PeopleEatChoice,
+};
+
+export default meta;
+
+export const Component: StoryObj<typeof PeopleEatChoice> = {
+    args: {
+        title: 'Europäisch',
```
[the remark this answers is not in the document]
Comment (Llavrov) [reply src/components/standard/choice/PeopleEatChoice.stories.tsx:13]: okay

[the remark this answers is not in the document]
Comment (CemYil03) [reply src/components/standard/choice/PeopleEatChoice.stories.tsx]: Not only here, but everywhere

**Review by CemYil03 — APPROVED** [review]

**Pull request merged**